CONTRIBUTING.md

```diff
@@ -117,18 +117,21 @@ For example:
 Within each test directory, `README.md` should document the test plan. The test
 name directory and the `*.go` files should be named after the test name as shown
 in the [project](https://github.com/orgs/openconfig/projects/2/views/1) item.
+
+Each test must also be accompanied by a `rundata_test.go` file that is
+automatically generated using the command `go run ./tools/addrundata --fix`. The
+rundata is for tracking the test result. If the rundata is outdated, the
+"Rundata Check" in the pull request check will fail.
```

Comment:
```suggestion
Each test must also be accompanied by a `rundata_test.go` file that is
automatically generated using the command `go run ./tools/addrundata --fix`. rundata_data
generates test results in junitxml format. If the rundata is outdated, the
"Rundata Check" in the pull request check will fail.
```

Would you mind adding link to a new a readme at tools/addrundata/readme.md with a quick explanation.  Others may find the test result output useful.

Reply:
Added tools/addrundata/README.md and moved detail there.